curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/remove-whitespace-from-start-and-end.english.md

```diff
@@ -23,8 +23,8 @@ Write a regex and use the appropriate string methods to remove whitespace at the
 tests:
   - text: <code>result</code> should equal to <code>"Hello, World!"</code>
     testString: assert(result == "Hello, World!");
-  - text: You should not use the <code>.trim()</code> method.
-    testString: assert(!code.match(/\.trim\(.*?\)/));
+  - text: Your solution should not use the <code>String.prototype.trim()</code> method.
```

Comment:
Might be a better idea to use the full form, `String.prototype.prim()`, earlier in the `Note` -  

> The .trim() method would work here, but you'll need to complete this challenge using regular expressions.

and then just use `.trim()` (_or you could go for the full form too_) in the test case - 

> Your solution should not use the String.prototype.trim() method.

Also, while we're at it, wasn't the general convention to use a space after **Note** followed by the text, unless the note spanned multiple lines...